Lesson4.ipynb

```diff
@@ -29,6 +33,1310 @@
     "\n",
     "Select **Lesson4.ipynb**"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Read csv file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What is csv file?\n",
+    "\n",
+    "A CSV is a comma separated values file which allows data to be saved in a table structured format. CSVs are a kind of spreadsheet same with Excel but with a .csv extension. Traditionally they take the form of a text file containing information separated by commas, hence the name.\n",
+    "\n",
+    "** Content in CSV Sample **\n",
+    "\n",
+    "The followings are the part of data from Biggest Loser:\n",
+    "\n",
+    "team_no,team_name,team_captain,team_member,2018-04-02,2018-04-03\n",
+    "\n",
+    "1,TBD,FALSE,1-1,11980,10437\n",
+    "\n",
+    "1,TBD,FALSE,1-2,22935,13399\n",
+    "\n",
+    "However, csv file is a table, as we are familiar with Excel, to make it more clear, we can have a comparison with Excel we generally use in our daily work, the first row is the coloumn name. Start from the second row, they are the specific information of each person.\n",
+    "\n",
+    "You can open csv file by textEditer, you'll see the content same as above.\n",
+    "If you can open it with Excel, then it has the same lookings as .xml file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## How to read csv file in Python? ##\n",
+    "\n",
+    "To read and analyse the data, I recommend Pandas Library to implement the data analysis.\n",
+    "\n",
+    "### What is Pandas ###\n",
+    "\n",
+    "Pandas stands for “Python Data Analysis Library”. According to the Wikipedia page on Pandas, “the name is derived from the term “panel data”, an econometrics term for multidimensional structured data sets.” But I think it’s just a cute name to a super-useful Python library!\n",
```

Comment:
We discussed that pandas cannot be used in lesson 4, and is a bonus for lesson 8. Lesson 4 should be using normal python csv read.